cli/ballerina-cli-module/src/main/java/org/ballerinalang/cli/module/TokenUpdater.java

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.ballerinalang.cli.module;
+
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+import com.sun.net.httpserver.HttpServer;
+import org.ballerinalang.cli.module.util.ErrorUtil;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import java.net.InetSocketAddress;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Paths;
+import java.util.Collections;
+
+/**
+ * This class has a service which updates the access token in the `Settings.toml` file.
+ */
+public class TokenUpdater {
+
+    private static PrintStream errStream = System.err;
+    private static PrintStream outStream = System.out;
+
+    private TokenUpdater() {
+    }
+
+    public static void execute(String settingsTomlFilePath) {
+        HttpServer server;
+        try {
+            server = HttpServer.create(new InetSocketAddress(9295), 0);
```

Comment:
Is it ok to use hard coded port here? What if it is already used?

Reply:
At the moment we are only returning an error message. Still couldn't come up with a fix for this.